examples/example-with-custom-asg-role/main.tf

```diff
@@ -17,7 +17,7 @@ terraform {
 # ---------------------------------------------------------------------------------------------------------------------
 resource "aws_iam_service_linked_role" "consul_asg_role" {
   aws_service_name = "autoscaling.amazonaws.com"
-  custom_suffix    = "test-consul-service-linked-role"
+  custom_suffix    = "${format("consul-slr-%s",replace(replace(timestamp(),"-",""),":",""))}"
```

Comment:
What about adding a `custom_suffix` variable to this example set to something simple by default (e.g., `test-consul-service-linked-role` as you had before) and then overriding it at test time with some name that includes the `UniqueId()` from Terratest?

Reply:
I'm new to terratest so will have to figure that one out; but sounds like a better fix overall...